accounts/migrations/0001_initial.py

```diff
@@ -0,0 +1,28 @@
+# Generated by Django 3.1.2 on 2020-10-25 15:54
```

Comment:
Migrations should be in your gitignore so they arent pushed to the repo :) 
- git rm rf <file_name>
- add exclusions to gitignore 
- ??
- profit